README.md

```diff
@@ -55,6 +55,7 @@ Supported keys of the map are:
 ## License
 
 Copyright © 2016 Coda Hale
+Copyright © 2017 Sentry
```

Comment:
No worries — appreciate the context.

As it happens, I'd be happy to transfer this repo to Sentry. I wrote this library for Skyliner, which is closing its doors next month, and don't anticipate using this (or Clojure) in production for the foreseeable future. It'd be a shame to let this stagnate, especially when folks like you are putting in work like this!